submit.go

```diff
@@ -273,35 +285,13 @@ func parseJSONRequest(w http.ResponseWriter, req *http.Request, reportDir string
 		}
 	}
 
-	// backwards-compatibility hack: current versions of riot-android
-	// don't set 'app', so we don't correctly file github issues.
```

Comment:
+1 to ripping this out, though ideally it would have been a separate PR. Never mind now.